classes/DirtEvent.sc

```diff
@@ -84,8 +84,9 @@ DirtEvent {
 
 		if (~unit == \c) { speed = speed * ~unitDuration * ~cps };
 
-		endSpeed = speed * (1.0 + (accelerate.abs.linexp(0.01, 4, 0.001, 20, nil) * accelerate.sign));
-		if(endSpeed.sign != speed.sign) { endSpeed = 0.0 }; // never turn back
+		// endSpeed = speed * (1.0 + (accelerate.abs.linexp(0.01, 4, 0.001, 20, nil) * accelerate.sign));
+		// if(endSpeed.sign != speed.sign) { endSpeed = 0.0 }; // never turn back
+		endSpeed = speed * (1.0 + accelerate);
```

Comment:
Well, by "depending on dur" I really meant only because SuperDirt implements the effect by calculating a target `endspeed` (independent of dur), and essentially works out the dur and \Delta speed per time to get there.  Classic Dirt instead calculates the \Delta speed per time (independent of dur) straight from `accelerate`, since it's barely even aware of dur.

In the end, given that `unit` throws a wrench into the whole thing anyway, I'm not sure that it matters too much precisely what the implementation is, as long as it "sounds right".  And my opinion is that constant Delta speed per time is the most relevant thing to "sounds right". `accelerate`'s interactions with `speed`, `unit`, `sustain`, `end`, etc, are probably not something most users have intuition or expectations for.

tlrdr; it's a dev version - I support keeping the implementation simple, let the interactions fall where they do and fix later if something super weird crops up